### PR TITLE
Cog Introduced

### PR DESCRIPTION
**Pull request by quillfires**
Cog introduced (to be used with your main/other bot)
[x] No breaking changes

### CODE REVIEW

**Comment by greg6775** [comment]
Thanks for the Pull Request. I'll review it tomorrow.

**Comment by quillfires** [comment]
alright..i've tested and fixed the bugs...

File: invite_tracker.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            pass
+
+    @commands.Cog.listener()
+    async def on_guild_remove(self, guild):
+        try:
+            self.invites.pop(guild.id)
+        except:
+            pass
+
+    # @commands.guild_only()
```
Comment (greg6775) [inline invite_tracker.py]: Do we need this part? If no, remove it please

**Review by greg6775 — CHANGES_REQUESTED** [review]

File: invite_tracker.py (hunk 1 above)
Comment (quillfires) [reply invite_tracker.py]: Done

**Comment by quillfires** [comment]
the other part is needed as when the bot leaves a guild or joins a guild it updates the invites

**Review by greg6775 — APPROVED** [review]

**Pull request merged**